src/pages/api/orgs/[orgId]/events.ics/index.ts

```diff
@@ -0,0 +1,74 @@
+import dayjs, { Dayjs } from 'dayjs';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import BackendApiClient from 'core/api/client/BackendApiClient';
+import { ZetkinEvent } from 'utils/types/zetkin';
+
+const timeStamp = (t: Dayjs): string => t.format('YYYYMMDDTHHmmss');
+const formatString = (str: string): string =>
+  str
+    .replaceAll('\n', '\\n')
+    .match(/.{1,75}/g)
+    ?.join('\n\t') ?? '';
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
```

Comment:
This is an "old-school" ([Pages router](https://nextjs.org/docs/pages)) API route. I think we should move it to the app router, and to make the URL `/o/<id>/events.ics` by moving this file to `src/app/o/[orgId]/events/route.ts` and refactoring this function to an [app-router API route](https://nextjs.org/docs/app/api-reference/file-conventions/route).

This also makes me realize that we should do the same thing for project pages (but then filter the events for that project), i.e. `/o/<id>/projects/<id>/events.ics`.

Reply:
Got it converted to a app-router API route, and split up so that it can be somewhat reused for projects :)

I think the current test error isn't related to my changes.

Reply:
Fixed in 791279eaafb52ae8200c1fd6d4da4a422b2a81d9.